example/fedora/minimal-40-aarch64.yaml

```diff
@@ -45,7 +37,6 @@ otk.define:
 
 otk.target.osbuild.qcow2:
   pipelines:
-    - otk.include: "osbuild/buildroot.yaml"
-    - otk.include: "osbuild/pipeline/tree.yaml"
+    - otk.include: "repository/${version}/repositories.yaml"
```

Comment:
I personally would move this otk.include before the pipeline, it does not matter for the output but feels more "correct" because the repository defines are not really part of a pipeline, more some setup to enable the pipelines to work (but people can have different opinions here I suppose).